pkgs/tools/security/genpass/default.nix

```diff
@@ -7,15 +7,15 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "genpass";
-  version = "0.5.1";
+  version = "0.5.14";
 
   src = fetchgit {
     url = "https://git.sr.ht/~cyplo/genpass";
```

Comment:
I think the latest version is [v0.5.27](https://git.cyplo.dev/cyplo/genpass/releases/tag/v0.5.27), since upstream moved to its own Gitea instance https://git.cyplo.dev/cyplo/genpass